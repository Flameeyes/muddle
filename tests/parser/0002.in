Can we read a simple directive?

Hello, I am ${/top/my-name-is}  .
