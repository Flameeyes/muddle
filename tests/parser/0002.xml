<?xml version="1.0" ?>

<top>
  <my-name-is>Fred</my-name-is>
</top>
