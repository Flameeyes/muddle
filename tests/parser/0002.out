Can we read a simple directive?

Hello, I am Fred  .
